kubernetes/adapter.go

```diff
@@ -89,7 +89,7 @@ type Ingress struct {
 	SSLPolicy        string
 	IPAddressType    string
 	LoadBalancerType string
-	WafWebACLId      string
+	WAFWebACLId      string
```

Comment:
nit: Should the name be `WAFWebACLID` (`Id -> ID`) to follow the go conventions?